website/docs/r/key_vault_managed_hardware_security_module_key_rotation_policy.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Key Vault"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_key_vault_managed_hardware_security_module_key_rotation_policy"
+description: |-
+  Manages a Managed HSM Key rotation policy.
+---
+
+# azurerm_key_vault_managed_hardware_security_module_key_rotation_policy
+
+Manages a Managed HSM Key rotation policy.
+
+## Example Usage
+
+```hcl
+resource "azurerm_key_vault_managed_hardware_security_module_key" "example" {
+  name           = "example-key"
+  managed_hsm_id = azurerm_key_vault_managed_hardware_security_module.example.id
+  key_type       = "EC-HSM"
+  curve          = "P-521"
+  key_opts       = ["sign"]
+}
+
+resource "azurerm_key_vault_managed_hardware_security_module_key_rotation_policy" "example" {
+  managed_hsm_key_id = azurerm_key_vault_managed_hardware_security_module_key.example.id
+  expire_after       = "P60D"
+  time_before_expiry = "P30D"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `expire_after` - (Required) Specify the expiration time of the key.
+
+* `managed_hsm_key_id` - (Required) The ID of the Managed HSM Key. Changing this forces a new Key Vault to be created.
+
+---
+
+* `time_after_creation` - (Optional) Specify the rotation time after creation in the format of ISO86061. Exactly one of `time_after_creation` or `time_before_expiry` should be specified.
```

Comment:
this is not clear, it should be explicit in what it is doing when
```suggestion
* `time_after_creation_to_rotate` - (Optional) Specify the rotation time after creation in the format of ISO86061. Exactly one of `time_after_creation` or `time_before_expiry` should be specified.
```

also is this ISO 8601 duration not ISO86061?